gnat2goto/driver/goto_utils.adb

```diff
@@ -442,4 +442,35 @@ package body GOTO_Utils is
       return False;
    end Has_GNAT2goto_Annotation;
 
+   function Integer_Constant_To_Expr
```

Comment:
This seems similar to `Build_Index_Constant`.

Reply:
I'll keep this function for now as I'm not convinced `Build_Index_Constant` is doing the right thing by just converting silently converting non-bitvector integers to 32 bit integers